src/rai_extensions/rai_open_set_vision/rai_open_set_vision/vision_markup/boxer.py

```diff
@@ -81,6 +81,7 @@ def get_boxes(
         box_threshold: float,
         text_threshold: float,
     ) -> list[Box]:
+        # TODO: move this to method, or use RAI canonical one
```

Comment:
Are you planning to resolve this TODO in this PR?

Reply:
No, in the next one.